src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -140,16 +160,62 @@ const EditShiftHours = ({
           setShowEditShiftHours(false);
         }
       },
+      true,
       handleServerValidationErrors
     );
   };
 
+  const getCombinedErrors = () => {
+    const existingErrors =
+      Object.keys(errors).length > 0 ? errors : summaryErrors;
+
+    const combinedErrors = { ...existingErrors };
+
+    if (clashingProperty === 'startAndEndTime') {
+      combinedErrors['shift-start-time'] = {
+        message: (
+          <TimeInputErrors
+            clashingProperty={clashingProperty}
+            clashes={clashingTimes}
+          />
+        ),
+      };
+      combinedErrors['shift-finish-time'] = {
+        message: '',
+      };
+    }
+
+    if (clashingProperty === 'startTime') {
```

Comment:
Shouldn't these be `else if` statements as the `clashingProperty` can only be one of those 3 values at a time..

Reply:
👍  done